doc/help/MERGE.md

```diff
@@ -0,0 +1,44 @@
+kdb-merge(1) -- Three-way merge of KeySets.
+================================
+## DESCRIPTION
+Does a three-way merge between keysets.  
+On success the resulting keyset will be saved to mergepath.  
+On unresolved conflicts nothing will be changed.  
+
+## USAGE
```

Comment:
Looking at quite a few other man pages, the SYNOPSIS always just seems to be all the flags that can be used for the command. For instance:

      SYNOPSIS
      apt-get [-asqdyfmubV] [-o=config_string] [-c=config_file] [-t=target_release] [-a=architecture] {update | upgrade |
               dselect-upgrade | dist-upgrade | install pkg [{=pkg_version_number | /target_release}]...  | remove pkg...  |
               purge pkg...  | source pkg [{=pkg_version_number | /target_release}]...  |
               build-dep pkg [{=pkg_version_number | /target_release}]...  |
               download pkg [{=pkg_version_number | /target_release}]...  | check | clean | autoclean | autoremove |
               {-v | --version} | {-h | --help}}